src/praisonai-agents/praisonaiagents/sandbox/manager.py

```
@@ -63,273 +51,77 @@ async def __aexit__(self, exc_type, exc_val, exc_tb):
-                "available": docker.is_available,
-                "description": "Isolated Docker containers",
-                "requires": ["docker"],
-            }
+            registry_cls = get_sandbox_registry()
+            registry = registry_cls.default()
+            for name in registry.list_names():
+                available = False
+                try:
+                    cls = registry.resolve(name)
+                    available = bool(cls().is_available)
+                except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)